rust/operator-binary/src/crd.rs

```diff
@@ -131,6 +136,61 @@ impl AutoTlsCa {
     }
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "camelCase")]
+pub enum CertificateKeyGeneration {
+    Rsa {
+        /// The amount of bits used for generating the RSA keypair. 
```

Comment:
nit: Use "key pair".

```suggestion
        /// The amount of bits used for generating the RSA key pair. 
```